crwatcher/watcher.go

```diff
@@ -65,7 +66,7 @@ type CRWatcher struct {
 //      don't notice the deletion until the subsequent re-list.
 type ResourceController interface {
 	ResourceAdded(resource *unstructured.Unstructured)
-	ResourceUpdated(oldResource, newResource *unstructured.Unstructured)
+	ResourceUpdated(oldResource, newResource *unstructured.Unstructured, newResourceAfterUpdate bool)
```

Comment:
I don't love that we are changing this interface. I kind of lean toward making the controllers smarter rather than doing this, or instead of calling Updated we call Added.

Reply:
Changed to calling Added.